mlflow/tracking/fluent.py

```diff
@@ -214,6 +280,17 @@ def log_param(key, value):
 
     :param key: Parameter name (string)
     :param value: Parameter value (string, but will be string-ified if not)
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        params = {"learning_rate": 0.01, "n_estimators": 10}
+
+        # Use context manager to create a new run and log parameters
```

Comment:
```suggestion

```
I think we can remove this because `mlflow.__init__.py` contains the exact same comment.

Reply:
Yes, agreed. And we have it at the top of this document too.